sentry-android-core/src/main/java/io/sentry/android/core/ManifestMetadataReader.java

```diff
@@ -107,6 +107,8 @@ final class ManifestMetadataReader {
 
   static final String IGNORED_ERRORS = "io.sentry.ignored-errors";
 
+  static final String ENABLE_AUTO_TRACE_ID_GENERATION = "io.sentry.enable-auto-trace-id-generation";
```

Comment:
```suggestion
  static final String ENABLE_AUTO_TRACE_ID_GENERATION = "io.sentry.traces.enable-auto-id-generation";
```
to better align with the `io.sentry.<feature>` pattern we have here already